Tests/OpusTests/OpusDecoderTests.swift

```diff
@@ -15,4 +15,24 @@ final class OpusDecoderTests: XCTestCase {
 			}
 		}
 	}
+
+
+	func test_CorruptedOpusData_ThrowsInvalidPacketError() throws {
+		guard let opusFormat = AVAudioFormat(opusPCMFormat: .int16, sampleRate: .opus16khz, channels: 1) else {
+			XCTFail("Unable to generate desired opusFormat")
+			return
+		}
+
+		let opusDecoder = try Opus.Decoder(format: opusFormat)
+		let corruptedOpusData = Data([191, 232, 174, 215, 224, 130, 236, 126, 177, 204, 165, 85, 230, 201, 43, 16, 207, 120, 223, 247, 59, 117, 41, 235, 55, 96, 78, 68, 7, 207, 184, 255, 254, 0, 0, 0, 0, 0, 0, 0])
```

Comment:
Maybe add a comment that describes how and why this is bad data?

Reply:
I'll have to ask about this, as the data was provided by someone else. Probably not until sometime during the next week (they are currently not reachable).